pihole

```diff
@@ -117,6 +117,32 @@ piholeEnable() {
     #Disable Pihole
     sed -i 's/^addn-hosts/#addn-hosts/' /etc/dnsmasq.d/01-pihole.conf
     echo "::: Blocking has been disabled!"
+    if [[ $# > 1 ]] ; then
+      if [ -x "$(command -v nohup)" ]; then
+        if [[ ${2} == *"s"* ]] ; then
+          echo "Seconds"
+          tt=${2%"s"}
+          echo "::: Blocking will be reenabled in ${tt} seconds"
+          echo "sleep ${tt}; pihole enable"
+          nohup bash -c "sleep ${tt}; pihole enable" </dev/null &>/dev/null &
+        elif [[ ${2} == *"m"* ]] ; then
+          tt=${2%"m"}
+          echo "::: Blocking will be reenabled in ${tt} minutes"
+          tt=$((${tt}*60))
+          echo "sleep ${tt}; pihole enable"
+          nohup bash -c "sleep ${tt}; pihole enable" </dev/null &>/dev/null &
+        else
+          echo "::: Unknown format for delayed reactivation of the blocking!"
+          echo "::: Example:"
+          echo ":::    pihole disable 5s    -    will disable blocking for 5 seconds"
+          echo ":::    pihole disable 7m    -    will disable blocking for 7 minutes"
+          echo "::: Blocking will not automatically be reenabled!"
+        fi
+      else
+        echo "::: Command 'nohup' has to be available for this feature."
```

Comment:
Quickly verified that is should always be available. Only some linux based routers might not have it. So we should be on the safe side here.